examples/exampleutil/exampleutil.go

```diff
@@ -21,3 +24,10 @@ func Print(s string, a any) {
 	}
 	fmt.Println(string(res))
 }
+
+func RandomPaymail() string {
+	seed := time.Now().UnixNano()
+	n := rand.New(rand.NewSource(seed)).Intn(500)
+	addr := fmt.Sprintf("john.doe.%dtest@4chain.test.com", n)
```

Comment:
I would add current timestamp (in some format) instead of random number. 
But 🤷, your way is also ok 👍

Reply:
Agree, I'll handle this during improving all examples after implementing missing APIs.